sass/elements/_button.scss

```
@@ -0,0 +1,6 @@
+button {
+    cursor: pointer;
+    color: inherit;
+    font-size: inherit;
+    font-family: inherit;
+}
```

```suggestion
}

```
Should be a final newline.